src/convertToObject.js

```diff
@@ -3,16 +3,35 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+function splitKey(string) {
+  return string.split(':')[0].trim();
+}
+
+function splitValue(string) {
+  return string.split(':')[1].trim();
+}
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = {};
+  const splitedStr = sourceString.split(';');
+
+  for (const string of splitedStr) {
+    if (splitKey(string) && splitValue(string)) {
+      cssProperties[splitKey(string)] = splitValue(string);
```

Comment:
you 2 times get the key and value

```suggestion
    const key = splitKey(string);
    if (key && splitValue(string)) {
      cssProperties[key] = splitValue(string);
```